internal/users/cache/testdata/golden/TestDeleteUser/Delete_existing_user

```diff
@@ -1,7 +1,11 @@
-GroupByID: {}
-GroupByName: {}
-GroupByUGID: {}
-GroupToUsers: {}
+GroupByID:
+    "11111": '{"Name":"group1","GID":11111,"UGID":"12345678"}'
```

Comment:
I think this test should have 2 users in each groups so that we see the user being removed from those groups, but the groups still exists.

Reply:
Isn't that exactly what the Delete_existing_user_keeping_other_users_intact test case tests?

Reply:
Possibly if there are common groups between the user being deleted and others. Maybe we should rename it then and/or add a comment to the test case?

Reply:
I'll do that

Reply:
Done